pkg/datasource/report_content.go

```diff
@@ -27,8 +27,8 @@ func (datasource *MsupplyEresDatasource) GetReportContent(scheduleID string) ([]
 	defer rows.Close()
 
 	for rows.Next() {
-		var ID, ScheduleID, DashboardID, Variables, Lookback string
-		var PanelID int
+		var ID, ScheduleID, DashboardID, Variables string
+		var PanelID, Lookback int
```

Comment:
@mark-prins The reason why @sworup changed Lookback to string is mentioned in this [comment](https://github.com/openmsupply/excel-report-email-scheduler/issues/76#issuecomment-1161315333)
Could you please have a look at this?

Reply:
thanks @Mai21 - I don't understand what the problem is from that comment.. if you could help explain? 🤷 
The reason that I changed it from a string is that the grafana structure expects an int, and passing a string results in problems running the report. I spent some time adding in conversions but the end result is changing to an int. that's working for me on v8.5

Reply:
Hmm, I see. I am wondering if Grafana v8.5 fixed the issue about `$__timeTo()` 

> Just did some testing and found that grafana expects epoch timestamp in microseconds, while the date conversion in our code at least is converting in seconds. Now epoch timestamp system could be seconds, miliseconds or microseonds but $__timeTo() method does not do the check and expect the time to be milliseconds hence putting in the oldest 1970s date time... this ends up getting blank values.

As my understanding, according to Sworup's investigation, `$__timeTo()` method used to expect `milliseconds` while our code was converting in seconds. And String From "now - 5m",  To "now" worked as we expected, that's why he changed it to String. And in PNG, Grafana v8.5 & this Lookback string version of the plugin are working...

But anyway if the int works properly and Grafana expects an int, there is no way not to go with int I suppose.